docs/guides/Orders/orders-feed.md

```diff
@@ -260,49 +237,38 @@ Here are two complete example bodies for the [Feed configuration response](https
-2. For each one of these ten events, it evaluates to which status the order was changed.
-3. If the change was to the `Ready for Handling` status, the integration gets the complete order data, records it in the ERP, updates the status in VTEX to `Preparing Delivery` (informing the start of the handling).
+1. It retrieves ten events from the feed.
+2. For each one of the ten events, it evaluates the status the order was changed to.
+3. If the order changed to the status `Ready for handling`, the integration gets the complete order data, records it in the ERP, and updates the status on VTEX to `Handling shipping` (which indicates the start of handling).
 4. Then, it commits the events to the Feed (removing them from the queue).
-5. After reading and removing these ten events from the list, the integration moves to the next ten events in the Feed and repeats the whole process.
+5. After reading and removing these ten events from the list, the integration moves to the following ten events in the feed and repeats the process.
 
 ![feedFlow-min](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/orders-feed-0.png)
 
-Access this [backoffice order integration guide](https://developers.vtex.com/docs/guides/erp-integration-set-up-order-integration) to learn more about how you can integrate VTEX’s order Feed with an ERP.
+Check the [back-office order integration guide](https://developers.vtex.com/docs/guides/erp-integration-set-up-order-integration) to learn more about how you can integrate the VTEX order feed with an ERP.
 
 ## Hook
 
-The Hook is a complement to the Feed, which allows integrations to consume order updates data in a different way. Instead of receiving events to form a queue that can be retrieved, the Hook sends the items automatically to an URL provided by the user in the Hook configuration.
-[block:callout]
-{
-  "type": "warning",
-  "body": "Since the Hook is a complement, [access](https://developers.vtex.com/vtex-rest-api/docs/feed-v3-1#access) follows the same principles described above for the Feed. This means each appKey can configure or access only one Hook. Different users that share one appKey access the same Hook. Therefore we recommend the configuration of one Hook per appKey per user, ensuring each user has access to an exclusive Hook."
-}
-[/block]
+Hook is a counterpart to Feed. It allows integrations to consume order update data differently. Instead of receiving events to form a queue that can be retrieved, a hook automatically sends the items to a URL provided by the user in the hook configuration. [block:callout]
+{ "type": "warning", "body": "Since Hook is a counterpart, [access](#access) follows the same principles described for Feed above. This means each appKey can configure or access only one hook. Different users that sharing the same appKey access the same hook. We recommend configuring one hook per appKey per user, ensuring that each user has access to their own hook." } [/block]
```

Comment:
```suggestion
Hook is a counterpart to Feed. It allows integrations to consume order update data differently. Instead of receiving events to form a queue that can be retrieved, a hook automatically sends the items to a URL provided by the user in the hook configuration. 

>⚠️ Since Hook is a counterpart, [access](#access) follows the same principles described for Feed above. This means each appKey can configure or access only one hook. Different users that sharing the same appKey access the same hook. We recommend configuring one hook per appKey per user, ensuring that each user has access to their own hook.
```